model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py

```diff
@@ -0,0 +1,188 @@
+from model_analyzer.config.generate.model_variant_name_manager import ModelVariantNameManager
+from model_analyzer.config.run.model_run_config import ModelRunConfig
+from model_analyzer.config.run.run_config import RunConfig
+from model_analyzer.perf_analyzer.perf_config import PerfAnalyzerConfig
+from model_analyzer.triton.model.model_config import ModelConfig
+from model_analyzer.triton.client.client import TritonClient
+from model_analyzer.device.gpu_device import GPUDevice
+from model_analyzer.config.input.config_command_profile import ConfigCommandProfile
+from model_analyzer.config.input.objects.config_model_profile_spec import ConfigModelProfileSpec
+from model_analyzer.result.result_manager import ResultManager
+from model_analyzer.result.run_config_measurement import RunConfigMeasurement
+from model_analyzer.record.metrics_manager import MetricsManager
+from model_analyzer.result.results import Results
+from model_analyzer.result.run_config_result import RunConfigResult
+
+from model_analyzer.constants import LOGGER_NAME, MAGNITUDE_DECAY_RATE
+from model_analyzer.config.input.config_defaults import DEFAULT_NUM_CONFIGS_PER_MODEL, \
+    DEFAULT_RUN_CONFIG_MIN_CONCURRENCY, DEFAULT_RUN_CONFIG_MAX_CONCURRENCY
+
+from copy import deepcopy
+
+import logging
+
+logger = logging.getLogger(LOGGER_NAME)
+
+
+class QuickPlusConcurrencySweepRunConfigGenerator(ConfigGeneratorInterface):
+    """
+    First run QuickRunConfigGenerator for a hill climbing search, then use 
+    Brute for a concurrency sweep of the default and Top N results
+    """
+
+    def __init__(self, search_config: SearchConfig,
+                 config: ConfigCommandProfile, gpus: List[GPUDevice],
+                 models: List[ConfigModelProfileSpec], client: TritonClient,
+                 result_manager: ResultManager,
+                 model_variant_name_manager: ModelVariantNameManager):
+        """
+        Parameters
+        ----------
+        search_config: SearchConfig
+            Defines parameters and dimensions for the search
+        config: ConfigCommandProfile
+            Profile configuration information
+        gpus: List of GPUDevices
+        models: List of ConfigModelProfileSpec
+            List of models to profile
+        client: TritonClient
+        result_manager: ResultManager
+            The object that handles storing and sorting the results from the perf analyzer
+        model_variant_name_manager: ModelVariantNameManager
+            Maps model variants to config names
+        
+        model_variant_name_manager: ModelVariantNameManager
+        """
+        self._search_config = search_config
+        self._config = config
+        self._gpus = gpus
+        self._models = models
+        self._client = client
+        self._result_manager = result_manager
+        self._model_variant_name_manager = model_variant_name_manager
+        self._rcg = None
+
+    def set_last_results(self, measurements):
```

Comment:
I tried to add typing here: ```measurements: List[Union[RunConfigMeasurement,None]]``` (like in quick search)

But got the following error that I couldn't figure out how to fix: ```../../model_analyzer/config/generate/quick_plus_concurrency_sweep_run_config_generator.py:92: error: "None" has no attribute "set_last_results"```

I'm pretty sure this is because I initialize `self._rcg to None`, but that seems like the correct thing to do. Any suggestions?

Reply:
I've fixed this